tests/pytorch/test_pytorch_model_export.py

```diff
@@ -769,6 +790,41 @@ def test_load_model_succeeds_when_data_is_model_file_instead_of_directory(
     )
 
 
+def state_dict_equal(state_dict1, state_dict2):
+    for key in state_dict1.items():
+        if key not in state_dict2:
+            return False
+        value1 = state_dict1[key]
+        value2 = state_dict2[key]
+        if isinstance(value1, torch.Tensor) and isinstance(value2, torch.Tensor):
+            if not torch.equal(value1, value2):
+                return False
+        elif state_dict1[key] != state_dict2[key]:
```

Comment:
Can we reuse `value1` and `value2` here?

Reply:
done . i have uploaded the newer version.